projects/shared-pipes/safe-html.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+@Pipe({
+  name: 'safeHtml'
+})
+export class SafeHtmlPipe implements PipeTransform {
+
+  constructor(private domSanitizer: DomSanitizer) {
+  }
+
+  transform(html: string): SafeHtml {
+    return this.domSanitizer.bypassSecurityTrustHtml(html);
+  }
```

Comment:
This is good, please check if it should be this or a directive, or just a function in the component which sets the value of a variable.

Reply:
We are using a pure pipe, so it is only called when Angular detects a change in the value or the parameters passed to the pipe and until `community.about` changes, the pipe won't run again